README.md

```diff
@@ -1 +1,15 @@
 # Vac.dev Website
+
+made and deployed with [logos site-builder](https://github.com/acid-info/logos-site-builder)
+
+# Continuous Deloyment
+
+- `staging` branch is deployed to [staging.vac.dev](https://staging.vac.dev)
+- `master` branch is deployed to [vac.dev](https://vac.dev)
+
+# Change Process
+
+1. Create a new working branch from `staging`: `git checkout staging; git checkout -b my-changes`;
+2. Proceed with changes, push to `origin` and open a Pull Request against `staging`;
```

Comment:
WDYT of making the `staging` process optional? Personally I'd prefer to open against `master`, but if people find benefit in using `staging` we can have that option too.

We are mostly writing posts here, not a whole lot can go wrong with a static site, the consequences are negligible and it is easy to fix-up.

Reply:
We could do this, but we would have to rebase `staging` on master after doing that to keep them in sync.
This might lead to conflicts if there are PRs on staging and master at the same time.

Reply:
Could also setup a github action that automerges master into staging on new master commits.

Reply:
yeah, master should always be source of truth, and most of our work here is gonna be write-ups